plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -149,7 +149,7 @@
                 <version>${project.version}</version>
                 <executions>
                     <execution>
-                        <phase>process-sources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
I have done the follwowing:
1. rm -rf on my .m2/repository directory
2. mvn clean install on che-dependencies
3. mvn -o clean install on che
This worked, except that I had to manually build core/che-core-db in online mode because it uses flyway, which is not installed in the che-dependencies project.

I have confidence that the change works.